src/windows.rs

```diff
@@ -249,8 +249,14 @@ impl Window {
             };
 
             if user32::RegisterClassW(&class) == 0 {
-                println!("Unable to register class, error {}", kernel32::GetLastError() as u32);
-                return None;
+                let err_code = kernel32::GetLastError() as u32;
```

Comment:
Can just do

```Rust
if kernel32::GetLastError() as u32 != 1410 {
  ...
}
```

Also keep brace on the first line (I try to have the code following the Rust guidelines for code https://github.com/rust-lang/rust/blob/master/src/doc/style/style/braces.md )